### ISSUE TITLE
listVersions 500 server error

### ISSUE THREAD

**Issue by jhpedemonte** [maqetta/maqetta#2384]
Trying to recreate Brad's scenario from #2333, I logged in as a guest and on load, got an error in the JS and server consoles:

```
2012-05-23 09:07:07.873:WARN::ERROR:  /maqetta/cmd/listVersions
java.lang.NullPointerException
    at maqetta.core.server.user.ReviewManager.getDesignerUser(Unknown Source)
    at maqetta.core.server.user.ReviewManager.getDesignerUser(Unknown Source)
    at maqetta.core.server.command.ListVersions.handleCommand(Unknown Source)
    at org.maqetta.server.DavinciCommandServlet.doGet(Unknown Source)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
    at org.maqetta.server.DavinciCommandServlet.service(Unknown Source)
    at javax.servlet.http.HttpServlet.service(HttpServlet.java:820)
    at org.eclipse.equinox.http.registry.internal.ServletManager$ServletWrapper.service(ServletManager.java:180)
```

This is with the latest successful build from this morning (up to commit 3f23ce36b9c376d148d4854d14874bc6f5a49b58).

**Comment by childsb** [comment]
i suspect this is the same issue as #2333

**Comment by childsb** [comment]
Really fixed in commit fa58fe5d49570eca03381174182bf6b3b52504f9

The problem occurred when guest users invited real users to a review.  Later when the guest users are removed, the real user still existed.  The guest users reviews were indexed for the real user, but didn't actually exist.
